jekyll/_cci2/sample-config.md

```diff
@@ -47,18 +76,250 @@ workflows:
       - test
 ```
 
-{% endraw %}
+### Sequential Workflow
 
-This example shows a concurrent job workflow where the `build` and `test` jobs run concurrently to save time. Refer to the [Workflows]({{ site.baseurl }}/2.0/workflows) document for complete details about orchestrating job runs with concurrent, sequential, and manual approval workflows.
+The configuration example below shows a sequential job workflow where the `build` job runs and then the `test` job runs once `build` has completed. Refer to the [Workflows]({{ site.baseurl }}/2.0/workflows) document for complete details about orchestrating job runs with concurrent, sequential, and manual approval workflows.
+
+This image shows the workflow view for the following configuration example, in which jobs run sequentially; one after the other:
+![Sequential Workflow Map]({{ site.baseurl }}/assets/img/docs/sequential-workflow-map.png)
+
+{:.tab.basic-sequential.Cloud}
```

Comment:
Is the only thing differentiating the Cloud version and the Server version the `version: 2.x` key? If so, at that point  it might just be worth having a single code sample for this block and setting it to 2.1 (and maybe just adding a comment that says ` # use "2.0" if you are using a server installatio of CircleCI`

Reply:
hmm yeah... The other difference is you need `version: 2` under workflows for 2.0 but not 2.1 🤔

Reply:
Ah right, I missed that.